docs/podman-login.1.md

```diff
@@ -37,6 +37,12 @@ Username for registry
 **--authfile**
 Path of the authentication file. Default is ${XDG_\RUNTIME\_DIR}/containers/auth.json
 
+**--cert-dir**
+Pathname of a directory containing TLS certificates and keys
```

Comment:
Day late and dollar short Sweeney reporting in.  Would be nice to have a mention of what the certs/keys are used for.  In Buildah we say:

Use certificates at path (*.crt, *.cert, *.key) to connect to the registry

I think adding "used to connect to the registry" would be good here and maybe in the usage.

Reply:
@TomSweeneyRedHat Could you open a PR with those messages.